docs/en/advanced_guides/models.md

```diff
@@ -1 +1,99 @@
 # Models
```

Comment:
I think it would be better to add some information about every part of a segmetor, like backbone neck decode_head etc.